spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /index and /show' do
+    it 'Should get all posts by user with id of 7' do
+      get '/users/7/posts'
+      expect(response).to have_rendered('index')
+      expect(response.body).to include 'Pagination'
+      expect(response).to have_http_status(:success)
+    end
+    it 'Should get the post with id of 1 by user with id of 7' do
+      get '/users/7/posts/1'
+      expect(response).to have_rendered('show')
+      expect(response.body).to include 'Post #1 by username'
```

Comment:
- [ ] Good job creating controllers specs 👏 . I've noticed that you have used a matcher provided by the Capybara testing framework, however as for this project you are supposed to use `rspec-rails` gem `render_template`. Kindly update that . Note that both methods are used to confirm that the expected views are being rendered in a Ruby on Rails application.

```ruby
expect(response).to render_template(:show)
```